pages/common/hg-add.md

```diff
@@ -0,0 +1,24 @@
+# hg add
+
+> Adds specified files to the staging area for the next commit in Mercurial.
+> See the `hg` page for additional information.
```

Comment:
This line should be in the hg.md page as "See the hg-add and hg-commit page for additional information"

Reply:
Cheers. 👍 I'll make that change.